app/src/pages/inside/profilePage/profilePage.jsx

```diff
@@ -35,32 +39,82 @@ const messages = defineMessages({
     id: 'ProfilePage.title',
     defaultMessage: 'User profile',
   },
+  profilePageProjectAssignmentTab: {
+    id: 'ProfilePage.projectAssignmentTab',
+    defaultMessage: 'PROJECT ASSIGNMENT',
+  },
+  profilePageProjectApiKeysTab: {
+    id: 'ProfilePage.apiKeys',
+    defaultMessage: 'API KEYS',
+  },
+  profilePageConfigurationExamplesTab: {
+    id: 'ProfilePage.configurationExamples',
+    defaultMessage: 'CONFIGURATION EXAMPLES',
+  },
+});
+
+const getDefaultLinkConfig = (profileRoute) => ({
+  type: USER_PROFILE_PAGE,
+  payload: {
+    profileRoute,
+  },
 });
+
+const getNavigationTabsConfig = (formatMessage) => ({
+  [PROJECT_ASSIGNMENT_ROUTE]: {
+    name: formatMessage(messages.profilePageProjectAssignmentTab),
+    link: getDefaultLinkConfig(PROJECT_ASSIGNMENT_ROUTE),
+    component: <AssignedProjectsBlock />,
+  },
+  [API_KEYS_ROUTE]: {
+    name: formatMessage(messages.profilePageProjectApiKeysTab),
+    link: getDefaultLinkConfig(API_KEYS_ROUTE),
+    component: <AccessTokenBlock />,
+  },
+  [CONFIG_EXAMPLES_ROUTE]: {
+    name: formatMessage(messages.profilePageConfigurationExamplesTab),
+    link: getDefaultLinkConfig(CONFIG_EXAMPLES_ROUTE),
+    component: <ConfigExamplesBlock />,
+  },
+});
+
+@connect((state) => ({
+  activeTab: adminProfileRouteSelector(state),
+}))
 @injectIntl
 @track({ page: PROFILE_PAGE })
 export class ProfilePage extends Component {
   static propTypes = {
     intl: PropTypes.object.isRequired,
+    activeTab: PropTypes.string,
+    dispatch: PropTypes.func,
+  };
+
+  static defaultProps = {
+    activeTab: PROJECT_ASSIGNMENT_ROUTE,
+    dispatch: () => {},
   };
 
   getBreadcrumbs = () => [{ title: this.props.intl.formatMessage(messages.profilePageTitle) }];
 
   render = () => (
     <PageLayout>
       <PageHeader breadcrumbs={this.getBreadcrumbs()} />
-      <PageSection>
-        <div className={cx('container')}>
-          <div className={cx('column')}>
-            <PersonalInfoBlock />
-            <AssignedProjectsBlock />
-            <LocalizationBlock />
-          </div>
-          <div className={cx('column')}>
-            <AccessTokenBlock />
-            <ConfigExamplesBlock />
+      <div className={cx('container')}>
+        <section className={cx('content-wrapper')}>
+          <div>
+            <article className={cx('section-wrapper')}>
```

Comment:
I'm not sure that `article` tag relevant here as we don't store block of text information here.